src/main/java/seedu/club/model/Model.java

```diff
@@ -156,8 +158,17 @@ void updateMember(Member target, Member editedMember)
     void deleteTask(Task taskToDelete) throws TaskNotFoundException, TaskCannotBeDeletedException;
 
     void updateFilteredTaskList(Predicate<Task> predicate);
-
     //@@author Song Weiyang
+    /**
+     * Changes the password of the member in that list
+     * @param username
+     * @param oldPassword
+     * @param newPassword
+     * @throws PasswordIncorrectException
+     */
+    void changePassword(String username, String oldPassword, String newPassword)
+            throws PasswordIncorrectException, DataToChangeIsNotCurrentlyLoggedInMember;
```

Comment:
since `DataToChangeIsNotCurrentlyLoggedInMember` is an exception, it should be something like "...Exception"